### ISSUE TITLE
How I Use?

### ISSUE THREAD

**Issue by ShylokVakarian** [dewiniaid/sigsolve#2]

**Comment by ghost** [comment]
You don't. It's broken.
At best, after installing all dependencies you will be presented a solution, but it will not click the tiles for you.

For anyone still willing to use this poorly documented "tool":
Install Python from Windows Store 
(Yes, store, cmd wil not recognize the normal installation  & I don't want to deal with PATH on my home computer)
Open CMD
pip install pyautogui pyscreenshot pillow numpy
cd to where you unzipped sigsolve
python main.py

Make sure your command prompt is not covering the game, as that prevents it from recognizing the game board.
(Again, it will **only** list the solution, but not execute it -no clicking whatsoever will occur-
Basically better off just solving yourself compared to following the instructions. You'll fail at half but atleast it's a lot faster.)
![sigsolve](https://user-images.githubusercontent.com/7262090/143296768-3fce8a95-a815-46b0-8c2b-837e16cc3f49.png)

Edit: Actually, if you're only doing it for the achievement; using SAM is faster.
If you're doing it for the story, it's all fluff and vague mentions of characters and houses we'll never see/hear.
The only upside from trying to get this to work, is that I finally realized why it's called "Sigma's Garden".
The Alchemist was called Sigma, and we play the game using marbles. Sigma's Marbles. Sigma Balls.
